kubernetes/ldapsync.yml.erb

```diff
@@ -0,0 +1,32 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: ldapsync
+spec:
+  schedule: "*/30 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: ldapsync
+              image: "docker.ocf.berkeley.edu/ldapsync:<%= version %>"
+              command: ["/bin/sh", "-c"]
+              args:
+                - chronic -e python3 /opt/ldapsync/google_groups.py
```

Comment:
This is fine for now, but we should find a way to make sure all the logs are stored by Kubernetes, and only alert on "bad" events. That way we can inspect logs later on if we have to (either with `kubectl` commands or with Loki later on).

Reply:
You can select which events (exceptions/warnings about people not being in ldap/info about people being added) are shown by the `--log-level` flag. I can `tee` to `/dev/tty` to make sure that k8s stores the output, too, or use `stdin2syslog`.